contacts24/cli.py

```diff
@@ -9,39 +13,49 @@
     get_contact_phone,
     get_upcoming_birthdays,
     load_contacts_book,
-    parse_input
+    parse_input,
 )
 from .errors import HELP_ERROR_MESSAGE
 
+init(autoreset=True)
+
+
+contacts = load_contacts_book()
+
+commands = {
+    "hello": lambda x: "How can I help you?",
+    "help": lambda x: f"Available commands: {HELP_ERROR_MESSAGE}",
+    "add": partial(add_contact, contacts=contacts),
+    "change-phone": partial(change_contact, contacts=contacts),
+    "change-email": lambda x: "In development",
+    "change-address": lambda x: "In development",
+    "phone": partial(get_contact_phone, contacts=contacts),
+    "add-birthday": partial(add_birthday, contacts=contacts),
+    "show-birthday": partial(get_contact_birthday, contacts=contacts),
+    "birthdays": partial(get_upcoming_birthdays, contacts=contacts),
+    "all-contacts": partial(get_all_contacts, contacts=contacts),
+    "add-note": lambda x: "In development",
+    "change-note": lambda x: "In development",
+    "find-note": lambda x: "In development",
+    "get-all-notes": lambda x: "In development",
+}
+
 
 def start():
-    contacts = load_contacts_book()
+    print(Fore.GREEN + "Welcome to the assistant bot! Type 'help' to see available commands.")
     try:
         while True:
```

Comment:
I think we should swap try and while. try-catch should be inside loop so that loop is not broken after exception